db/migrations/20231012154828_add_ext_rss_feeds.js

```diff
@@ -0,0 +1,10 @@
+exports.up = function (knex) {
+  return knex.schema.createTable("external_feed", function (table) {
+    table.uuid("guid").primary().unique();
+    table.text("feed_url").notNullable();
```

Comment:
```suggestion
    table.text("title").notNullable();
```

We can probably drop this now right? Or we could replace with a human readable `title` so it's easy to tell what the record is for